ci/test_conda.sh

```diff
@@ -6,19 +6,15 @@ set -euo pipefail
 . /opt/conda/etc/profile.d/conda.sh
 
 rapids-logger "Install testing dependencies"
-# TODO: Replace with rapids-dependency-file-generator
-rapids-mamba-retry create -n test \
-    c-compiler \
-    cxx-compiler \
-    cuda-nvcc \
-    cuda-nvrtc \
-    cuda-python \
-    cuda-version=${RAPIDS_CUDA_VERSION%.*} \
-    "numba>=0.58" \
-    make \
-    psutil \
-    pytest \
-    python=${RAPIDS_PY_VERSION}
+
+ENV_YAML_DIR="$(mktemp -d)"
+
+rapids-dependency-file-generator \
+  --output conda \
+  --file_key test \
```

Comment:
```suggestion
  --file-key test \
```

Could you change these to `--file-key` (hyphen instead of underscore)?

Per https://github.com/rapidsai/dependency-file-generator/issues/89.

That'd avoid this warning: https://github.com/rapidsai/pynvjitlink/actions/runs/9358256535/job/25759883265?pr=86#step:7:25

> /opt/conda/lib/python3.9/site-packages/rapids_dependency_file_generator/_cli.py:94: UserWarning: The use of --file_key is deprecated. Use -f or --file-key instead.
  warnings.warn("The use of --file_key is deprecated. Use -f or --file-key instead.")

Reply:
Done in ed7b254! Ready for another review.